webui/react/src/pages/JobQueue/JobQueue.tsx

```diff
@@ -117,6 +117,10 @@ const JobQueue: React.FC = () => {
         }
       },
     };
+    if (process.env.IS_DEV && isJobOrderAvailable &&
```

Comment:
**question:** just confirming we still want to limit this to dev builds only?

Reply:
yup we wouldn't have backend support for this part in master yet. I want this code to be present in case the backend support landed when I was OOO so there is no webui work